cernopendata/modules/pages/views.py

```diff
@@ -111,6 +111,7 @@ def md_debug():
 
 
 @blueprint.route('/visualise/events')
+@blueprint.route('/visualize/events')
```

Comment:
please check https://github.com/cernopendata/opendata.cern.ch/blob/master/cernopendata/modules/pages/views.py#L270 so as to avoid extra lines :)